templates/lxc-archlinux.in

```diff
@@ -107,6 +108,11 @@ sed -e 's/^ConditionPathExists=/# ConditionPathExists=/' \
 sed -e 's/^ConditionPathIsReadWrite=\/proc\/sys\/$/ConditionPathIsReadWrite=\/proc\/sys\/net\//' \
     -e 's/^ExecStart=\/usr\/lib\/systemd\/systemd-sysctl$/ExecStart=\/usr\/lib\/systemd\/systemd-sysctl --prefix net/' \
     -i /usr/lib/systemd/system/systemd-sysctl.service
+# fixes for systemd in lxc container
+ln -s /dev/null /etc/systemd/system/systemd-udevd.service
+ln -s /dev/null /etc/systemd/system/systemd-udevd-control.socket
+ln -s /dev/null /etc/systemd/system/systemd-udevd-kernel.socket
+ln -s /dev/null /etc/systemd/system/proc-sys-fs-binfmt_misc.automount
```

Comment:
Per mailing-list comment, all of those have conditions not to start in containers and so shouldn't be needed.